packages/ng/forms/multilanguage-input/multilanguage-input.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="textField">
+	<div class="textField-input">
+		<input
+			luInput
+			type="text"
+			class="textField-input-value"
+			[placeholder]="placeholder()"
+			[(ngModel)]="invariant().value"
+			(ngModelChange)="valueChange()"
+			(blur)="onTouched?.()"
+			[disabled]="disabledInternal()"
+			#inputElement
+		/>
+		<div class="textField-input-affix">
+			<button
+				[luPopover2]="popoverMultilanguage"
+				[customPositions]="popoverPositions"
+				[luTooltip]="intl.toggleMultilanguage"
+				type="button"
+				class="button mod-onlyIcon mod-text textField-input-affix-toggle"
+			>
+				<span aria-hidden="true" class="lucca-icon icon-signTranslation"></span>
+			</button>
+			<ng-template #popoverMultilanguage>
+				<div class="popover-contentOptional" [style.width.px]="inputElement.clientWidth + 16">
+					<div class="u-displayFlex u-gapXS u-flexDirectionColumn">
+						@for (row of panelInputs(); track row.cultureCode) {
+						<lu-form-field [label]="intl.translateTo | intlParams: {lang: getLocaleDisplayName(row.cultureCode)}" hiddenLabel>
+							<lu-text-input
+								[suffix]="{content: row.cultureCode, ariaLabel: ''}"
+								[(ngModel)]="row.value"
+								(ngModelChange)="valueChange()"
+								[disabled]="disabledInternal()"
+							></lu-text-input>
```

Comment:
Use a [consistent ordering of attributes](https://github.com/angular-eslint/angular-eslint/blob/main/packages/eslint-plugin-template/docs/rules/attributes-order.md)?

```suggestion
		<input
			#inputElement
			class="textField-input-value"
			type="text"
			luInput
			[disabled]="disabledInternal()"
			[placeholder]="placeholder()"
			[(ngModel)]="invariant().value"
			(ngModelChange)="valueChange()"
			(blur)="onTouched?.()"
		/>
		<div class="textField-input-affix">
			<button
				class="button mod-onlyIcon mod-text textField-input-affix-toggle"
				type="button"
				[customPositions]="popoverPositions"
				[luPopover2]="popoverMultilanguage"
				[luTooltip]="intl.toggleMultilanguage"
			>
				<span class="lucca-icon icon-signTranslation" aria-hidden="true"></span>
			</button>
			<ng-template #popoverMultilanguage>
				<div class="popover-contentOptional" [style.width.px]="inputElement.clientWidth + 16">
					<div class="u-displayFlex u-gapXS u-flexDirectionColumn">
						@for (row of panelInputs(); track row.cultureCode) {
						<lu-form-field [label]="intl.translateTo | intlParams: {lang: getLocaleDisplayName(row.cultureCode)}" hiddenLabel>
							<lu-text-input
								[disabled]="disabledInternal()"
								[suffix]="{content: row.cultureCode, ariaLabel: ''}"
								[(ngModel)]="row.value"
								(ngModelChange)="valueChange()"
							></lu-text-input>
```